tests/main_test.py

```diff
@@ -24,7 +27,26 @@
         'mock_func': ['reply_text'],
         'mock_ret': [True],
         'is_async': True
+    },
+    {
+        'func': main.generate_welcome,
+        'expected_res': 'Benvenuto @user nel nostro gruppo ^-^', 
```

Comment:
Maybe loading the `welcome.json` at the begging of the test and getting the first element is better.